content/library/advanced-features/advanced-widget-behavior.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: Widget semantics
-slug: /library/advanced-features/widget-semantics
+title: Widget behavior
+slug: /library/advanced-features/widget-behavior
 ---
 
-# Advanced notes on widget behavior
+# Understanding widget behavior
```

Comment:
We don't define on this page "what is a widget?" - that is, which streamlit commands does this page refer to?

Not sure if it's defined clearly somewhere else - either add it here, or link to that definition. Could be simple / abstract version similar to the TLDR

Reply:
I can expand the intro to include a definition. Good point.